RecoTracker/MkFit/plugins/MkFitProducer.cc

```diff
@@ -491,6 +495,14 @@ std::vector<const DetLayer *> MkFitProducer::createDetLayers(const mkfit::LayerN
   return dets;
 }
 
+bool MkFitProducer::passCCC(const SiStripRecHit2D& hit, const DetId hitId) const {
+  return (siStripClusterTools::chargePerCM(hitId,hit.firstClusterRef().stripCluster()) < 1620 );
```

Comment:
Should have been
```cpp
  return (siStripClusterTools::chargePerCM(hitId,hit.firstClusterRef().stripCluster()) >= 1620 );
```
Lesson learned: don't merge anything without a round of validation.